app/mailers/submission_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class SubmissionMailer < ApplicationMailer
+  def sentMsg(submission)
+    @submission = submission
+    mail(from: 'asd@w.pl', to:'asdf@w.pl', subject: 'Informacja o zmianie statusu')
```

Comment:
Tutaj dane muszą być poprawne:
* `from` można pominąć - bo jest ustawione domyślnie
* `to` powinien wskazywać mejl autora korespondującego
* `subject` powinno raczej brzmieć - "Decyzja - Rocznik Kognitywistyczny"